library/src/main/java/com/google/android/exoplayer/util/ManifestFetcher.java

```diff
@@ -143,11 +143,13 @@ public void updateManifestUrl(String manifestUrl) {
    * @param callbackLooper The looper associated with the thread on which the callback should be
    *     invoked.
    * @param callback The callback to receive the result.
+   * @return Instance of SingleFetchHelper which is loading the manifest.
    */
-  public void singleLoad(Looper callbackLooper, final ManifestCallback<T> callback) {
+  public SingleFetchHelper singleLoad(Looper callbackLooper, final ManifestCallback<T> callback) {
```

Comment:
I don't see how you can actually use this, because SingleFetchHelper is a private inner class. We don't want to make it public, either.